Resources/Prototypes/Recipes/Crafting/Graphs/improvised/flowerwreath.yml

```diff
@@ -0,0 +1,26 @@
+- type: constructionGraph
+  id: flowerwreath
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: flowerwreath
+          steps:
+            - tag: Flower
+              name: flower
+              icon:
+                sprite: Objects/Specific/Hydroponics/poppy.rsi
+                state: produce
+            - tag: Flower
+              name: flower
+              icon:
+                sprite: Objects/Specific/Hydroponics/poppy.rsi
+                state: produce
+            - tag: Ambrosia
+              name: ambrosia
+              icon:
+                sprite: Objects/Specific/Hydroponics/ambrosia_vulgaris.rsi
+                state: produce
+              doAfter: 10
+    - node: flowerwreath
+      entity: ClothingHeadHatFlowerWreath
```

Comment:
i thought this was ClothingNeckFlowerWreath

Reply:
oh yeah my mistake when i coped it over